cronjob

```diff
@@ -1 +1 @@
-* * * * * root /usr/local/bin/npx ts-node /usr/src/app/src/get-latest-quotes.ts >> /var/log/cron.log 2>&1
+* * * * * root . /etc/environment; /usr/local/bin/npx ts-node /usr/src/app/src/get-latest-quotes.ts >> /var/log/cron.log 2>&1
```

Comment:
also having the env vars is super important. not sure why the cron job was not getting access to them from the docker setup but this works.

i suspect the CRLF line ending issue never made it into git due to how git saves files but on my local system it was a major source of confusion.